x/group/spec/05_client.md

```diff
@@ -0,0 +1,557 @@
+<!--
+order: 6
+-->
+
+# Client
+
+## CLI
+
+A user can query and interact with the `group` module using the CLI.
+
+### Query
+
+The `query` commands allow users to query `group` state.
+
+```bash
+simd query group --help
+```
+
+#### group-info
+
+The `group-info` command allows users to query for group info by given group id.
+
+```bash
+simd query group group-info [id] [flags]
+```
+
+Example:
+
+```bash
+simd query group group-info 1
+```
+
+Example Output:
+
+```bash
+admin: cosmos1..
+group_id: "1"
+metadata: null
+total_weight: "3"
+version: "1"
+```
+
+#### group-account-info
+
+The `group-account-info` command allows users to query  for group account info by group account address
+
+```bash
+simd query group group-account-info [group-account] [flags]
+```
+
+Example:
+
+```bash
+simd query group group-account-info cosmos1..
+```
+
+Example Output:
+
+```bash
+address: cosmos1..
+admin: cosmos1..
+decision_policy:
+  '@type': /cosmos.group.v1beta1.ThresholdDecisionPolicy
+  threshold: "1"
+  timeout: 1s
```

Comment:
let's have an example timeout that could actually fit to real-life use cases (1s is quite a small amount of time to be able to vote on a proposal! :p)